app/lending/(1-concepts)/collateralisation/page.mdx

```diff
@@ -0,0 +1,103 @@
+
+If a user maintains an excessively low LTV (high overcollateralisation), a gradually increasing **Low LTV Penalty** is applied. This penalty is **0% at 50% LTV** and increases **linearly up to 1% APR as LTV approaches 0%**.  
+This discourages locking up unnecessary collateral that reduces system-wide economic security.
+
+Chainflip supports multi-collateral accounts. The oracle value of all supported collateral assets is combined to determine total collateral value for LTV calculation.
+
+---
+
+### Oracle Safety Controls
+
+To protect borrowers and the system:
+
+- Liquidations halt if oracle prices become stale
+- New loan creation halts while oracles are stale
+- Borrowers may still repay, add collateral, or close positions during these pauses
+
+Future versions will support additional safety mechanisms like taking Chainflip's own DEX  prices into account.
+
+---
+
+### Auto-Collateralisation
+
+Borrowers may enable auto-top-up, which uses free balance in the State Chain account in their asset of choice to automatically increase collateral when LTV approaches the soft liquidation threshold. Auto-top-up prevents liquidation as long as the borrower has sufficient available balance. Note that for the Beta launch, Auto Top up will be disabled.
+
+---
+
+### Liquidation Overview
+
+Chainflip performs liquidations directly through its own cross-chain DEX. This provides two major advantages over traditional lending platforms:
+
+- Support for Patrial Liquidations: The protocol liquidates only what's necessary to bring LTV back to a safe range
+- Execution is native: LPs fill liquidation orders directly, removing the need for external liquidators demanding steep discounts
+
+<Callout type="info">
+Chainflip does not use external liquidators.
+All liquidation orders are filled natively by LPs on the DEX, using the same quoting logic they use for regular swaps.
+This removes the need for specialist liquidators or discounted auctions found in other lending protocols.
+</Callout>
+
+Liquidations begin when LTV crosses soft or hard liquidation thresholds.
+
+---
+### Soft Liquidation
+
+Soft liquidation is triggered when **collateralisation** drops below the soft threshold.
+
+The protocol submits small DCA-style liquidation orders with a minimum price slightly below the oracle value using Live Price Protection (LPP). LPs compete to fill these orders at near-market rates.
+
+Liquidation continues until:
+
+- the loan becomes healthy
+- the loan is fully repaid
+- collateral is exhausted
+
+Borrowers may also manually trigger soft liquidation to unwind a position and reclaim remaining collateral.
+
+> **Important:**
+> - When soft liquidation is triggered automatically, the system only swaps small chunks until LTV returns to healthy levels.
+> - When a borrower triggers manual liquidation, the entire collateral balance is swapped. Any excess remains in the collateral balance.
```

Comment:
Can we somehow make this more prominent? 

This is how it looks right now:
<img width="892" height="258" alt="image" src="https://github.com/user-attachments/assets/9978f2c2-061d-4ffc-a77b-6789a2041082" />